docs/data/joy/components/table/TableSortAndSelection.tsx

```diff
@@ -390,7 +374,8 @@ export default function TableSortAndSelection() {
           rowCount={rows.length}
         />
         <tbody>
-          {stableSort(rows, getComparator(order, orderBy))
+          {[...rows]
+            .sort(getComparator(order, orderBy))
```

Comment:
As @Janpot mentioned, sorted rows should be memoized (here and in other demos).

Reply:
Do you mind if that change is raised separately? If you prefer, then can be added to this PR, but doesn’t seem related to IE 11

Reply:
> Do you mind if that change is raised separately?

OK, we can do it this way.

> if you merely memoise sorted rows, the slice() and map() are going to produce new React elements anyway.

Yes, but the array won't be unnecessarily sorted on each render.